packages/core/vitest-integration.setup.ts

```diff
@@ -0,0 +1,25 @@
+import { beforeEach } from "vitest"
+import { PostgreSqlContainer, type StartedPostgreSqlContainer } from "@testcontainers/postgresql"
+import { createMigrator, createKysely } from "@dotkomonline/db"
+import { createEnvironment } from "@dotkomonline/env"
+
+const containers: StartedPostgreSqlContainer[] = []
+
+beforeEach(async () => {
+  const container = await new PostgreSqlContainer("public.ecr.aws/z5h0l8j6/dotkom/pgx-ulid:0.1.3")
+    .withExposedPorts(5432)
+    .withUsername("local")
+    .withPassword("local")
+    .withDatabase("main")
+    .start()
+  process.env.DATABASE_URL = container.getConnectionUri()
+  const env = createEnvironment()
+  const kysely = createKysely(env)
+  const migrator = createMigrator(kysely, new URL("node_modules/@dotkomonline/db/src/migrations", import.meta.url))
+  await migrator.migrateToLatest()
+  containers.push(container)
+})
+
+process.on("exit", async () => {
```

Comment:
If this code is expected to run, I believe the event that should be listened to is "beforeExit". No additional work will be done after the "exit" event, so the container.stop calls will not run.

See https://nodejs.org/api/process.html#event-exit and https://nodejs.org/api/process.html#event-beforeexit

Reply:
Ahh good catch